python/libraries/websocket.py

```diff
@@ -22,6 +24,7 @@
 connect_options_type = {
     "onOpen": (send_type, n_cmd_type.with_typevars(["bool"])),
     "onMessage": (send_type, "str", n_cmd_type.with_typevars(["bool"])),
+    "runAlways": (send_type, close_type, n_cmd_type.with_typevars(["unit"]))
```

Comment:
Hmm, are we ignoring backwards compatibility here? Because old code will not have specified `runAlways` and would encounter a type error when upgrading

Reply:
We could just like... remove this and make the standard to make everything that runs in runAlways just go in onConnect, because it has a send and an effective close function, but it would still need changes in the code. Backwards compatibility would be preserved tho.